src/dbnode/storage/series/lookup/entry.go

```diff
@@ -229,11 +229,6 @@ func (entry *Entry) LoadBlock(
 	block block.DatabaseBlock,
 	writeType series.WriteType,
 ) error {
-	// TODO(bodu): We can remove this once we have index snapshotting as index snapshots will
```

Comment:
As discussed looks like this is going to have be enabled during peer bootstrapping but not during commit log snapshot restoration.

Reply:
Added a separate `LoadBlockAndIndex()` API that performs indexing. 

Was less awkward than stringing through options to enable indexing to the `*lookup.Entry`.